tests/benchmark_test.py

```diff
@@ -135,6 +135,26 @@ def test_start_project_from_scratch(mock_collect_user_input):
     assert result.stdout.strip() == expected_output.strip()
 
 
+def import_module_from_path(module_name, relative_file_path):
+    """
+    Imports a from the file Mentat just edited.
+    import in Python is always relative to the file's original path so
+    even though we change the cwd, we can't use relative imports to import from the tmp dir.
+
+    I don't really know what module_name is for since it even works as the empty string.
```

Comment:
The module name is the name of the actual module you're importing; i.e. 'collections', 'itertools', so on. We should just make testbed the default module_name (so it's almost like we're writing 'from testbed import relative_file_path'). Otherwise we stand the risk of somehow overwriting a module with an empty string as it's name (unlikely, but still good practice I think).